examples/simulation/Tutorial4/macros/plots.C

```diff
@@ -29,8 +31,8 @@ plots(Int_t nEvents = 1000, Int_t iout = 1, TString mcEngine = "TGeant3")
     TFile *f1 = TFile::Open(MCFile);
     TFile *f2 = TFile::Open(RecoFile);
 
-    TTree *t1 = f1->Get("cbmsim");
-    TTree *t2 = f2->Get("cbmsim");
+    TTree *t1 = (TTree *)f1->Get("cbmsim");
```

Comment:
Please use C++ casts and not the C one. 
Or even better use the templated Get function

TTree* t1    = f1->Get<TTree>("cbmsim");

Reply:
Maybe?

```c++
auto t1 = f1->Get<TTree>("cbmsim");
```

Or maybe
```c++
std::unique_ptr<TTree> t1{f1->Get<TTree>("cbmsim")};
```